.github/workflows/pypi_build_and_images.yaml

```diff
@@ -0,0 +1,39 @@
+# This workflow will build and upload WMCore core services to the production PYPY
+# based on tag releases
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+  # for manual builds on GH Action page please uncomment the following section
+#   workflow_dispatch:
```

Comment:
As we discussed yesterday, please enable manual trigger of the this workflow.

Reply:
done